tests/datadriven/test_variants.py

```diff
@@ -192,6 +193,14 @@ def _verifyVariantCalls():
                     self.assertEqual(str(alt1), str(alt2))
             else:
                 self.assertEqual(gaVariant.alternate_bases, alt)
+            if not gaVariant.filters_applied:
```

Comment:
Are you confident these tests do run for some of our example data? There isn't a good way to guarantee this path is hit per test, since not all VCF will have filters. Maybe we could guarantee that for the suite this path is hit? If there is a file that demonstrates this behavior in the tests we might mention it in a comment for now.

The compliance VCF has all `PASS` I think https://github.com/ga4gh/compliance/blob/master/test-data/brca1_1kgPhase3_variants.vcf .